### ISSUE TITLE
compile instructions not working on rpi - Missing headers on build

### ISSUE THREAD

**Issue by jonnypage** [retuxx/hid-retrobit#4]
Trying to install the drivers on retropie 3.4, and I get this error when I'm trying to compile the .deb

```
sudo dpkg -i hid-retrobit.deb
(Reading database ... 64904 files and directories currently installed.)
Preparing to unpack hid-retrobit.deb ...

------------------------------
Deleting module version: 1.0.0
completely from the DKMS tree.
------------------------------
Done.
Unpacking hid-retrobit (1.0.0) over (1.0.0) ...
Setting up hid-retrobit (1.0.0) ...

Creating symlink /var/lib/dkms/hid-retrobit/1.0.0/source ->
                 /usr/src/hid-retrobit-1.0.0

DKMS: add completed.
Error! echo
Your kernel headers for kernel 4.1.17-v7+ cannot be found at
/lib/modules/4.1.17-v7+/build or /lib/modules/4.1.17-v7+/source.
Error! echo
Your kernel headers for kernel 4.1.17-v7+ cannot be found at
/lib/modules/4.1.17-v7+/build or /lib/modules/4.1.17-v7+/source.
modprobe: FATAL: Module hid-retrobit not found.
dpkg: error processing package hid-retrobit (--install):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 hid-retrobit

```

**Comment by msteinbeck** [comment]
Did you install the kernel headers? Unfortunately, I don't have a rpi and, thus, can't test hid-retrobit on it.

**Comment by jonnypage** [comment]
After a lot of trying different things, I found I had to install dpkg, an older version of gcc (4.7, apparently it isn't fully backwards compatible, or the check wasn't for newer versions) and then manually download specific headers before I could install the headers, and then install the driver. It did work at the end, but there are some serious hoops to jump through.

Thanks for the drivers though. Work great once I was able to get them working!
